.github/workflows/isotovideo-check-all-test-modules.yml

```diff
@@ -13,4 +13,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
```

Comment:
https://github.com/os-autoinst/os-autoinst-distri-example/pull/31#discussion_r1483187814

> Can you also change the action to actions/checkout@v4?

Reply:
Sure. i was thinking about that. I just do not know where to find the latest version

Reply:
https://github.com/actions/checkout

Reply:
Done